prime-router/src/main/kotlin/Element.kt

```diff
@@ -237,10 +238,12 @@ data class Element(
     /**
      * A formatted string is the Element's normalized value formatted using the format string passed in
      * The format string's value is specific to the type of the element.
+     * The schema is passed in for logging purposes only.
```

Comment:
Change this comment to 
The [schema] is passed ...
This will generate the proper kdoc